turbinia/workers/analysis/jenkins.py

```diff
@@ -163,7 +163,8 @@ def analyze_jenkins(version, credentials):
     credentials_registry = {hash: username for username, hash in credentials}
     # TODO: Add timeout parameter when dynamic configuration is ready.
     # Ref: https://github.com/google/turbinia/issues/244
-    weak_passwords = bruteforce_password_hashes(credentials_registry.keys())
+    weak_passwords = bruteforce_password_hashes(
+        credentials_registry.keys(), tmp_dir=None, extra_args='-m 3200')
```

Comment:
Can we pass in `self.tmp_dir` as the default here as well?  Are there any other invocations of this method that we need to update as well?

Reply:
There is no self here, as it's a static method.

Reply:
No other invocations as far as I can see.

Reply:
Ah, right.   Ok, thanks.